assets/functions/packer/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "scripts": {
+    "build": "npx webpack --config webpack.config.js"
```

Comment:
npx feels npm specific, what about yarn users? 

wouldn't it be better to remove npx here and use the local dep?

Reply:
Yeah, fair point. The packer project is an isolated project which is "invisible" for the pro-coder. So in this case, it is acceptable.